vcf/gVcf_test.go

```diff
@@ -1,26 +0,0 @@
-package vcf
-
```

Comment:
Why is this test file removed? From my end it looks like the gVcf.go file will still be around.

Reply:
GVCF is being largely removed in this PR. What's left is a few functions so that we don't break older programs that still used GVCF instead of VCF, but ultimately GVCF should be removed entirely IMO.